src/main/kubectl.ts

```diff
@@ -98,9 +100,22 @@ export class Kubectl {
     this.path = path.join(this.dirname, binaryName)
   }
 
-  public async getPath(): Promise<string> {
+  public getBundledPath() {
+    return Kubectl.bundledKubectlPath
+  }
+
+  public async getPath(bundled = false): Promise<string> {
+    // return binary name if bundled path is not functional
+    if (!await this.checkBinary(this.getBundledPath(), false)) {
+      Kubectl.invalidBundle = true
+      return path.basename(bundledPath)
```

Comment:
What happens if `kubectl` is not in PATH?

Reply:
I've added `KubeAuthProxy` to log error if spawning the process fails now.

Reply:
Ok, I think we need to improve this in an another PR. User should see this kind of error directly (because nothing works if kubectl is broken).